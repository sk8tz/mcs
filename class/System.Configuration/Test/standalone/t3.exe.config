<configuration>

<appSettings>
	<add key="hithere" value="Hello World!"/>
	<add key="hithere" value="Goodbye World!"/>
</appSettings>

</configuration>
