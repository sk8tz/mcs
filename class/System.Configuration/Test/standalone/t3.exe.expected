IsMachineLevel: False
settings[hithere] = Goodbye World!
AppSettings[hithere] = Goodbye World!
